### PR TITLE
feat: easier setup

### PR DESCRIPTION
**Pull request by alaeddine-13**
getting rid of the need to create folders and pass bootstrap parameter

### CODE REVIEW

File: jina/orchestrate/deployments/__init__.py [hunk 1]
```diff
@@ -144,14 +144,19 @@ def join(self):
         def wait_start_success(self):
             for pod in self._pods:
                 pod.wait_start_success()
-            if self._pods[0].args.stateful and self._pods[0].args.raft_bootstrap:
+            # TODO: is this the right behavior ?
+            # should this be done only when the cluster is started ?
+            if self._pods[0].args.stateful:
```
[the remark this answers is not in the document]
Comment (JoanFM) [reply jina/orchestrate/deployments/__init__.py:149]: does it affect if I add an already-existing voter?

File: jina/serve/consensus/run.go [hunk 2]
```diff
@@ -342,3 +360,66 @@ func add_voter(self *C.PyObject, args *C.PyObject) *C.PyObject {
     C.Py_IncRef(C.Py_None);
     return C.Py_None;
 }
+
+//export get_configuration
+func get_configuration(self *C.PyObject, args *C.PyObject) *C.PyObject {
```
Comment (JoanFM) [inline jina/serve/consensus/run.go:365]: does not passed the `header` file declaration

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

File: jina/serve/consensus/run.go [hunk 3]
```diff
@@ -342,3 +349,74 @@ func add_voter(self *C.PyObject, args *C.PyObject) *C.PyObject {
...
+
+        if len(conf.Servers) == 0 {
+            C.Py_IncRef(C.Py_None);
+            return C.Py_None;
+        }
+
+
+        server := findServerByID(conf.Servers, raft.ServerID(C.GoString(raftId)))
+        cstr := C.CString(string(server.Address))
+        defer C.free(unsafe.Pointer(cstr))
```
[the remark this answers is not in the document]
Comment (alaeddine-13) [reply jina/serve/consensus/run.go:414]: will attempt to remove it and update

[the remark this answers is not in the document]
Comment (JoanFM) [reply jina/serve/consensus/run.go:414]: let's do in separate PR, let's go for next milestone here

**Pull request merged**